src/sentry/api/event_search.py

```diff
@@ -1404,12 +1404,20 @@ def resolve_orderby(orderby, fields, aggregations):
             validated.append(prefix + bare_column)
             continue
 
-        if bare_column in FIELD_ALIASES and FIELD_ALIASES[bare_column].get("column_alias"):
+        if (
+            bare_column in FIELD_ALIASES
+            and FIELD_ALIASES[bare_column].get("column_alias")
+            and bare_column != PROJECT_ALIAS
+        ):
             prefix = "-" if column.startswith("-") else ""
             validated.append(prefix + FIELD_ALIASES[bare_column]["column_alias"])
             continue
 
-        found = [col[2] for col in fields if isinstance(col, (list, tuple))]
+        found = [
+            col[2]
+            for col in fields
+            if isinstance(col, (list, tuple)) and col[2].strip("`") == bare_column
```

Comment:
Pretty sure we need this check here otherwise you can order by unselected fields if histogram or project transform is included.